addons/Media_Recorder/src_webpack/MediaRecorder.jsm

```diff
@@ -327,11 +345,9 @@ export class MediaRecorder {
         DOMOperationsUtils.showErrorMessage(view, Errors, validatedModel.fieldName.join("|") + "_" + validatedModel.errorCode);
     }
 
-    setVisibility(isVisible) {
-        this.addonViewService.setVisibility(isVisible);
-    }
-
     _showBrowserError(view) {
-        DOMOperationsUtils.showErrorMessage(view, {unsupportedBrowser: "Your browser is not supported."}, "unsupportedBrowser");
+        let $wrapper = $(view).find(".media-recorder-wrapper");
+        $wrapper.addClass("media-recorder-wrapper-browser-not-supported");
+        $wrapper.text("Your browser is not supported.");
```

Comment:
Kody błędów generalnie dobrze by było trzymać w oddzielnej strukturze, tak jak masz to np. w Hierarchical_Lesson_Report. Takie rozwiązanie byłoby spójne z pozostałymi addonami, i ułatwiałoby edycję i przejrzystość komunikatów, zwłaszcza w wypadku pojawienia się nowych komunikatów w przyszłości. Natomiast jest to raczej mała kwestia, więc raczej poprawiałbym to tylko jeśli będziesz coś jeszcze zmieniał.

Reply:
poprawiono, treść przeniesiono do Errors.js